Makefile

```diff
@@ -150,6 +151,9 @@ platform-unixlike:
 platform-android-arm64:
 	make GOOS=android GOARCH=arm64 NPMDIR=npm/esbuild-android-arm64 platform-unixlike
 
+platform-android-arm:
+	make GOOS=android GOARCH=arm NPMDIR=npm/esbuild-android-arm platform-unixlike
```

Comment:
I am unable to run this on my computer. I get this:

```
$ make GOOS=android GOARCH=arm NPMDIR=npm/esbuild-android-arm platform-unixlike
test -n "android" && test -n "arm" && test -n "npm/esbuild-android-arm"
mkdir -p "npm/esbuild-android-arm/bin"
cd "npm/esbuild-android-arm" && npm version "0.12.25" --allow-same-version
v0.12.25
CGO_ENABLED=0 GOOS="android" GOARCH="arm" go build "-ldflags=-s -w" -trimpath -o "npm/esbuild-android-arm/bin/esbuild" ./cmd/esbuild
# github.com/evanw/esbuild/cmd/esbuild
loadinternal: cannot find runtime/cgo
/usr/local/go/pkg/tool/darwin_amd64/link: running clang failed: exit status 1
ld: warning: option -s is obsolete and being ignored
ld: unknown option: -z
clang: error: linker command failed with exit code 1 (use -v to see invocation)

make: *** [platform-unixlike] Error 2
```

This is on macOS 11.5.1. I don't understand why cgo is apparently enabled despite CGO_ENABLED=0. Here is a reduced reproduction:

```
$ cat main.go 
package main

import "runtime/debug"

func main() {
	debug.SetGCPercent(-1)
}

$ CGO_ENABLED=0 GOOS="android" GOARCH="arm" go build main.go
# command-line-arguments
loadinternal: cannot find runtime/cgo
/usr/local/go/pkg/tool/darwin_amd64/link: running clang failed: exit status 1
ld: unknown option: -z
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

This appears to be a bug in Go.

Reply:
Did you try compiling with older Golang versions?

Don't know why but I feel it's somehow related to wrong dynamic linker path when cross compiling issue (not mentioning the issue here since it may be unrelated)

Reply:
The Go team said that "internal linking" is unsupported with Android ARM and building this requires a C toolchain to build. Maybe this means I need to build this on a 32-bit Android ARM device?

Are you able to build esbuild using this command? If so, what OS version, CPU architecture, and Go version are you on?

Reply:
I was able to build on Android ARM 7.0 using on Termux using
```
go build ./cmd/esbuild
```
and I have a `esbuild` binary.

And about cross compiling, you need to get the C compiler from Android NDK.